PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -322,3 +325,13 @@ def miniAOD_customizeAllMC(process):
     miniAOD_customizeCommon(process)
     miniAOD_customizeMC(process)
     return process
+
+def miniAOD_forPhase2(process):
+    print "Switching off electron and photon IDs (not yet working for Phase2 era)"
+    process.patElectrons.addElectronID = False
+    process.patElectrons.electronIDSources = cms.PSet()
+    process.patPhotons.addPhotonID = False 
+    process.patPhotons.photonIDSources = cms.PSet()
```

Comment:
Is the underlying difficulty that miniAOD and PAT are using the same module declarations but want different behaviors for Phase2? What about making miniAOD an era? I believe that is what we did for fast sim. Then there is no need to worry about running the era and a customize together since there will only be the era.